setupext/develop.py

```diff
@@ -27,7 +27,20 @@ class Develop(develop_cmd):
         ('enable-coverage', None, 'Instrument c++ code for code coverage measuring'),
     ]
 
+#<<<<<<< HEAD:setupext/dist.py
```

Comment:
merge conflict here.

Reply:
Got it.  I guess i missed testing that one.  There were a lot of conflicts when the setup got changed.